### DISCUSSION TITLE
Replacement for @bentley/imodelhub-client in 3.0

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#2992] [Q&A] [answered]
I cannot compile my application because I cannot find any replacement for bentley/imodelhub-client in 3.0. But when I install this package, it is asking for other packages from 2.19 such as bentley/bentleyjs-core (which has been repaced by itwin/core-bentley). 
Kindly help me understand the workaround for it.

**Comment by aruniverse** [comment, marked as answer]
That package will be removed in 3.0 in favor of @itwin/imodels-access-frontend and @itwin/imodels-client-management.
